abixen-platform-core/src/main/java/com/abixen/platform/core/domain/service/AclEntryService.java

```diff
@@ -26,6 +28,10 @@
 
     List<AclEntry> findAll(final AclClassName aclClassName, final Long objectId);
 
+    List<AclEntry> findAll(PermissionName permissionName, AclSidType aclSidType, Long sidId, AclClassName aclClassName, Long objectId);
+
+    List<AclEntry> findAll(PermissionName permissionName, AclSidType aclSidType, List<Long> sidIds, AclClassName aclClassName, Long objectId);
+
     AclEntry create(AclEntry aclEntry);
```

Comment:
Interface should have final parameters in method to inform developer what he should implement method. Missing final